README.md

```diff
@@ -61,6 +73,26 @@ TCMALLOC_RELEASE_RATE 	default: 1.0 	Rate at which we release unused memory to t
 
 Note: This is typically set in the roslaunch file for the planning server node leveraging the `<env name="TCMALLOC_RELEASE_RATE" value="10">` tag.
 
+### Default Ubuntu OS malloc (ptmalloc)
+
+The great thing is, the default malloc library `ptmalloc` used provided run-time [tunables](https://www.gnu.org/software/libc/manual/html_node/Tunables.html), leveraging a set of environment variables.
+
+The one parameter that has been used to configure libc so it always gives back the memory to the system is setting following memory tunnable:
```

Comment:
You should elaborate on why giving memory back to the system is important and why it doesn't do it by default.

Also `tunnable` -> `tunable`

Reply:
I believe the main paragraph under `Process Planning Server Tuning` explains this, does it not?

See below:

The planning server is built leveraging the library taskflow which enables you to quickly write parallel and heterogeneous tasks programs which aligns well with process motion planning. These types of libraries leverage thread pools to quickly spin up different tasks as needed. One interesting thing found, if a task consumes a lot of memory libc will make a performance decision to allow the thread to hold onto the memory it has for subsequent tasks opposed to giving the memory back to your system. This may be fine if this was the only set of tasks being run but typically in our use case there are other computationally intensive processes which could use this memory.

Reply:
Okay I think that's fine. I must have skipped over that section

Reply:
Yea it most likely was not in the changes because I don't think I changed this paragraph.